packages/upload-client/src/sharding.js

```diff
@@ -84,3 +88,30 @@ export class ShardingStream extends TransformStream {
     })
   }
 }
+
+/**
+ * Default comparator for FileLikes. Sorts by file name in ascending order.
+ *
+ * @param {FileLike} a
+ * @param {FileLike} b
+ * @param {(file: FileLike) => string} getSortKey
+ */
+export const defaultFileComparator = (a, b, getSortKey = (a) => a.name) => {
```

Comment:
Minor, but `getSortKey` is actually returning a value not a key.

Reply:
ah I was using `key` in the sense of this https://docs.python.org/3/howto/sorting.html#key-functions
but I think you mean it like the 'key/value' of the File object, which is probably a more reasonable interpretation in this context than what I meant. I will rename it to `getSortValue` because that seems at least slightly less likely to cause confusion than what I did